xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -384,9 +384,20 @@ For more information, see [Exclude all null value properties](/dotnet/standard/s
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Determines whether read-only fields are ignored during serialization.
+            A property is read-only if it isn't marked with the <c>readonly</c> keyword.
+            The default value is <see langword="false" />.</summary>
         <value>To be added.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if read-only fileds should be ignored during serialization; <see langword="false" /> otherwise.</value>
```